src/client/components/Navigation/Topnav.js

```diff
@@ -299,36 +307,48 @@ class Topnav extends React.Component {
 
   handleClickMenu = e => this.setState({ selectedPage: e.key });
 
+  handleScroll  = () => {
+    const { prevScrollpos } = this.state;
+
+    const currentScrollPos = window.pageYOffset;
+    const visible = prevScrollpos < currentScrollPos;
+
+    this.setState({
+      prevScrollpos: currentScrollPos,
```

Comment:
Save prev position in instance (this.prevScrollpos = currentScrollPos;) - not in state